opendistro-for-elasticsearch/hooks/init

```diff
@@ -0,0 +1,39 @@
+#!/bin/sh
+
+mkdir -p {{pkg.svc_path_config_path}}/tmp
+mkdir -p {{pkg.svc_data_path}}/{{cfg.es_yaml.cluster.name}}
+
+#This section makes sure that the admin key exists.
+
+#if [ ! -s {{pkg.svc_config_path}}/certificates/MyRootCA.key ]; then
```

Comment:
Why is this commented?  I forget the cases in which `init` runs (does it run on upgrade?). But, if we don't need it, let's delete the commented code.